.storybook/main.ts

```diff
@@ -6,23 +7,30 @@ const config: StorybookConfig = {
 		disableTelemetry: true,
 	},
 
-	docs: {
-		autodocs: true,
-	},
+	docs: {},
 
 	framework: {
-		name: "@storybook/react-vite",
+		name: getAbsolutePath("@storybook/react-vite"),
 		options: {
 			strictMode: true,
 		},
 	},
 
-	stories: [
-		"../packages/react/src/**/*.mdx",
-		"../packages/react/src/**/*.stories.@(js|jsx|mjs|ts|tsx)",
+	stories: ["../packages/react/src/**/*.stories.{ts,tsx,mdx}"],
```

Comment:
This prevents Storybook from recognizing the new [Multiple Choice docs](https://github.com/wwnorton/design-system/blob/deps-upgrade-v1.9.0/packages/react/src/patterns/MultipleChoice/MultipleChoice.mdx) as a story.

I changed this in #366 to support [custom docs](https://storybook.js.org/docs/writing-docs/mdx#setup-custom-documentation) for Multiple Choice since it has two top-level components and the default docs only support one. This is what they recommend:

```suggestion
	stories: [
		"../packages/react/src/**/*.mdx",
		"../packages/react/src/**/*.stories.@(js|jsx|mjs|ts|tsx)",
	],
```

Reply:
nice catch! I resolved a conflict in this file and removed it without noticing. Thanks!